hosts_example.yml

```diff
@@ -21,6 +21,13 @@ all:
     ## For SASL/GSSAPI uncomment this line and see Kerberos Configuration properties below
     # sasl_protocol: kerberos
 
+    ## Zookeeper server-to-server and client-to-server DIGEST-MD5 authentication
+    ## Do NOT MIX it with sasl_protocol: kerberos
+    # zookeeper_sasl_digestmd5_enabled: true
```

Comment:
I'm thinking there should be a variable `zookeeper_sasl_protocol: kerberos/digestmd5`

Reply:
I'd suggest to use a similar `listener` approach as the kafka_brokers. Just not support for multiple listeners. Seeing SSL support is available in ZK as well and would allow for the SSL additions to keep to a common config style we have going on.

Reply:
I currently don’t see any reasons to encrypt connection between kafka brokers and zookeeper. 
1) Still Impossible to encrypt zk 2 zk members  connection. This is by-design
2) This PR still allows read-only access to anyone
3) There is nothing to hide there